django_emarsys/service.py

```diff
@@ -0,0 +1,25 @@
+from django_emarsys import api
```

Comment:
I'm not so sure about that one. `service` might not be the best choice, but to name the module after the model seems confusing.

I'll keep this in mind.